InfoScreen.c

```diff
@@ -109,11 +109,19 @@ void InfoScreen_run(InfoScreen* this) {
          MEVENT mevent;
          int ok = getmouse(&mevent);
          if (ok == OK) {
-            if (mevent.y >= panel->y && mevent.y < LINES - 1) {
-               Panel_setSelected(panel, mevent.y - panel->y + panel->scrollV);
-               ch = 0;
-            } else if (mevent.y == LINES - 1) {
-               ch = IncSet_synthesizeEvent(this->inc, mevent.x);
+            if (mevent.bstate & BUTTON1_RELEASED) {
+               if (mevent.y >= panel->y && mevent.y < LINES - 1) {
+                  Panel_setSelected(panel, mevent.y - panel->y + panel->scrollV - 1);
+                  ch = 0;
+               } else if (mevent.y == LINES - 1) {
+                  ch = IncSet_synthesizeEvent(this->inc, mevent.x);
+               }
+            #if NCURSES_MOUSE_VERSION > 1
+            } else if (mevent.bstate & BUTTON4_PRESSED) {
+               ch = KEY_WHEELUP;
+            } else if (mevent.bstate & BUTTON5_PRESSED) {
+               ch = KEY_WHEELDOWN;
+            #endif
             }
```

Comment:
Maybe avoid syntax injection inside code block changing its structure.
```suggestion
            }
            #if NCURSES_MOUSE_VERSION > 1
            else if (mevent.bstate & BUTTON4_PRESSED) {
               ch = KEY_WHEELUP;
            } else if (mevent.bstate & BUTTON5_PRESSED) {
               ch = KEY_WHEELDOWN;
            }
            #endif
```

Reply:
I've followed the code style from ScreenManager.c